### PR TITLE
feat(zh-hk.yml): Added Traditional Chinese language

### PR DESCRIPTION
**Pull request by chunkiuu**
Changes: 
Added Traditional Chinese language `zh-hk.yml`

I have ensured all translations are correct, as I am a native speaker of the Traditional Chinese language. :D

### CODE REVIEW

File: waypoints/src/main/resources/lang/zh-hk.yml [hunk 1]
```diff
@@ -0,0 +1,714 @@
+# 使用格式: https://docs.advntr.dev/minimessage/format.html
+#Translations made by ItzTheBear (Discord: chunkiu), please contants me if there are typo or incorrect translations :D
+#本翻譯是由 ItzTheBear 創建 (Discord: chunkiu) 請聯絡本人如果有任何翻譯錯誤，或您可以自行在Github上進行Pull request。
```
Comment (Sytm) [inline waypoints/src/main/resources/lang/zh-hk.yml]: Please make this header more like the others, although it is fine by me if you add your Discord in there

File: waypoints/src/main/resources/lang/zh-hk.yml [hunk 2]
```diff
@@ -0,0 +1,714 @@
...
+      # required_level: Level required for teleportation
+      xp: |
+        <red>你沒有足夠的等級用於傳送.
+        <red>需要等級: <yellow><required_level>
+      xpPoints: |
+        <red>你沒有足夠的經驗值用於傳送.
+        <red>你當前的經驗值: <yellow><current_points>
+        <red>需要經驗值: <yellow><required_points>
+        # current_balance: Vault balance of the player
+        # required_balance: Vault balance required for teleportation
```
Comment (Sytm) [inline waypoints/src/main/resources/lang/zh-hk.yml]: The indentation is wrong, these are considered part of the string above

File: waypoints/src/main/resources/lang/zh-hk.yml [hunk 3]
```diff
@@ -0,0 +1,714 @@
...
+    # waypoint: Name of the waypoint
+    waypoint: "<dark_gray>座標 <gold><waypoint>"
+    playerTracking: "<dark_gray>玩家追蹤"
+    sharing:
+      # waypoint: Name of the waypoint
+      selectPlayer: "<dark_gray>選擇玩家以分享 <gold><waypoint></gold>"
+      sharedWaypoints: "<dark_gray>與你共享的座標"
+      # waypoint: Name of the waypoint
+      sharedWith: "<dark_gray>你與以下玩家共享的 <gold><waypoint></gold>"
+    confirm: "<dark_gray>你確定嗎？"
```
Comment (Sytm) [inline waypoints/src/main/resources/lang/zh-hk.yml:286]: Trailing space

File: waypoints/src/main/resources/lang/zh-hk.yml [hunk 4]
```diff
@@ -0,0 +1,714 @@
...
+          <gray>距離: <gold><distance:'.0'>
+      public:
+        displayName: "<aqua><name>"
+        description: *waypointDescription
+        customDescription: *waypointCustomDescription
+      permission:
+        displayName: "<light_purple><name>"
+        description: *waypointDescription
+        customDescription: *waypointCustomDescription
+        owner: "<gray>擁有者: <gold><owner>"
```
Comment (Sytm) [inline waypoints/src/main/resources/lang/zh-hk.yml]: This is supposed to be a child of public, not permission

File: waypoints/src/main/resources/lang/zh-hk.yml [hunk 5]
```diff
@@ -0,0 +1,714 @@
...
+      particle:
+        name: "粒子指南針"
+        description: |
+          <gray>你腳下的粒子顯示座標所在的方向
+      trail:
+        name: "粒子軌跡"
+        description: |
+          <gray>計算並顯示通往座標的
+          <gray>可能路徑, 用粒子軌跡顯示
+  # amount: Amount of waypoints in this global folder visible for the player
```
Comment (Sytm) [inline waypoints/src/main/resources/lang/zh-hk.yml:437]: Empty newline above went missing (I like to keep the spacing between entries across languages identical)

**Review by Sytm — CHANGES_REQUESTED** [review]
Thank your very much for this contribution!

Although there are some tiny issues that should be resolved quickly

**Comment by chunkiuu** [comment]
I have made a corrected version `zh-hk.yml`
Sorry for having too many errors. The indentation problems should be fixed. Now it should be exactly the same as the `en.yml`.
Let me know if there are more mistakes that need to be fixed :D

**Review by Sytm — APPROVED** [review]

**Pull request merged**